src/libraries/System.Text.Json/gen/Resources/Strings.resx

```diff
@@ -130,7 +130,7 @@
     <value>Did not generate serialization metadata for type.</value>
   </data>
   <data name="ContextClassesMustBePartialMessageFormat" xml:space="preserve">
-    <value>Derived 'JsonSerializerContext' type '{0}' specifies JSON-serializable types. The type and all containing types must be made partial to kick off source generation.</value>
+    <value>Derived 'JsonSerializerContext' type '{0}' and all containing types must be made partial to kick off source generation.</value>
```

Comment:
Suggest rewording to avoid "kick off source generation":
"..and all containing types must be made partial for source generation."